admin/lib/errors/mtc-check-window-not-found.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
```

Comment:
the messages being separate from the error type feels fragmented.  Could we not house the message as a const within the type its used by?  seems a bit overkill to have a separate file for the message.  It should have only one purpose.